shared-libs/memdown-medic/src/memdown-medic.js

```diff
@@ -4,7 +4,7 @@
  *
  * USAGE
  * 	memdownMedic = require('memdown-medic');
- * 	memdownMedic('webapp/src')
+ * 	memdownMedic('./')
```

Comment:
I suspect this could/should just be `'.'`.

Reply:
Fixed. The default value is `./` so I suspect it just works.